### PR TITLE
Add configuration option for HiveMQ listener names

### PR DESCRIPTION
**Pull request by afalhambra-hivemq**
https://hivemq.kanbanize.com/ctrl_board/22/cards/26671/details/

### CODE REVIEW

File: charts/hivemq-platform/tests/configuration/hivemq_rest_api_listeners_configuration_test.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+suite: HiveMQ Platform Configuration - REST API listeners tests
```
Comment (Donnerbart) [inline charts/hivemq-platform/tests/configuration/hivemq_rest_api_listeners_configuration_test.yaml:1]: Should we have a failing test when someone tries to use `hivemqListenerName` with `type: rest-api`?

File: charts/hivemq-platform/tests/configuration/hivemq_control_center_listeners_configuration_test.yaml [hunk 2]
```diff
@@ -0,0 +1,570 @@
+suite: HiveMQ Platform Configuration - Control Center listeners tests
```
Comment (Donnerbart) [inline charts/hivemq-platform/tests/configuration/hivemq_control_center_listeners_configuration_test.yaml:1]: Should we have a failing test when someone tries to use `hivemqListenerName` with `type: control-center`?

**Review by Donnerbart — APPROVED** [review]
Very nice cleanup with the test class split. That's much easier to read!

File: charts/hivemq-platform/tests/configuration/hivemq_rest_api_listeners_configuration_test.yaml (hunk 1 above)
Comment (afalhambra-hivemq) [reply charts/hivemq-platform/tests/configuration/hivemq_rest_api_listeners_configuration_test.yaml:1]: Nice catch!

File: charts/hivemq-platform/tests/configuration/hivemq_control_center_listeners_configuration_test.yaml (hunk 2 above)
Comment (afalhambra-hivemq) [reply charts/hivemq-platform/tests/configuration/hivemq_control_center_listeners_configuration_test.yaml:1]: Nice catch!

**Review by mhofsche — APPROVED** [review]

**Pull request merged**